importexport/musicxml/importmxmlpass2.cpp

```diff
@@ -910,42 +911,50 @@ static void handleTupletStop(Tuplet*& tuplet, const int normalNotes)
 //   addArticulationToChord
 //---------------------------------------------------------
 
-static void addArticulationToChord(ChordRest* cr, SymId articSym, QString dir)
+static void addArticulationToChord(ChordRest* cr, SymId articSym, QString dir, QString place)
       {
       Articulation* na = new Articulation(articSym, cr->score());
-      if (dir == "up") {
-            na->setUp(true);
-            na->setAnchor(ArticulationAnchor::TOP_STAFF);
-            na->setPropertyFlags(Pid::ARTICULATION_ANCHOR, PropertyFlags::UNSTYLED);
+
+      if (!dir.isNull()) {
+            na->setUp(dir.isEmpty() || dir == "up");
+            if (dir.isEmpty()) // Set as an implicit value
```

Comment:
Now that its use has increased, there is a lot of duplicated code in the property flag handling. Suggest creating a few small convenience functions.